cmd/format.go

```diff
@@ -201,8 +201,12 @@ func format(c *cli.Context) error {
 	}
 
 	blob, err := createStorage(&format)
-	if err != nil && !noCheck {
-		logger.Fatalf("object storage: %s", err)
+	if err != nil {
+		if noCheck {
+			logger.Warnf("object storage: %s", err)
```

Comment:
blob is nil, it will crash later.

I think we should disable the check in ceph